qiskit_nature/problems/second_quantization/lattice/__init__.py

```diff
@@ -0,0 +1,53 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
```

Comment:
For example, how about `lattice_model/lattices/lattice.py`? Or maybe `lattice_problem/lattices/lattice.py`?

Reply:
This is something for @pbark to decide

Reply:
I believe the best way to go is `qiskit_nature/problems/second_quantization/lattice/lattices/lattice.py` . This will follow the plural convention and at the same time will be self - explanatory.